pkg/playbook/app/vcertutil/vcertutil.go

```diff
@@ -69,7 +73,6 @@ func EnrollCertificate(config domain.Config, request domain.PlaybookRequest) (*c
 		zap.L().Debug("successfully requested certificate", zap.String("requestID", reqID))
 
 		vRequest.PickupID = reqID
-		vRequest.Timeout = 180 * time.Second
```

Comment:
If you remove this line, and the first call to retrieve fails (certificate is not in the ISSUED state), you will have to keep invoking the retrieve function or fail. The timeout in the request object is to loop over the certificate retrieval for slow CAs.

Reply:
Do not remove it. It makes sense.

Reply:
It doesn't make sense, that timeout is already set before